UM/Scene/Scene.py

```diff
@@ -175,10 +175,14 @@ def _onFileChanged(self, file_path: str) -> None:
         try:
             if os.path.getsize(file_path) == 0:  # File is empty.
                 return
-        except EnvironmentError:  # Or it doesn't exist any more, or we have no access any more.
+        except EnvironmentError:  # Or it doesn't exist anymore, or we have no access anymore.
+            return
+
+        # On Mac the reload file dialog would pop up for recently downloaded files, even if the file did not change
```

Comment:
```suggestion
        # Fixes CURA-9417
        # On Mac the reload file dialog would pop up for recently downloaded files, even if the file did not change
```

Thought it would be handy to mention the ticket here in case the comes up again.